locales/en.yml

```diff
@@ -81,6 +81,7 @@ en:
           keynote_name:
           keynote_description:
         keynote_about: "?"
+    call-to-speakers: "Posit your talk until June 30"
```

Comment:
This would be better as "Apply with your talk proposal until June 30". The ">" character is also missing while present on the Spanish version.